test/e2e/fixtures.go

```diff
@@ -146,10 +147,16 @@ func setupTestWithIPFIXCollector(tb testing.TB) (*TestData, error, bool) {
 	ipfixCollectorIP, err := testData.podWaitForIPs(defaultTimeout, "ipfix-collector", testNamespace)
 	if err != nil || len(ipfixCollectorIP.ipStrings) == 0 {
 		tb.Errorf("Error when waiting to get ipfix collector Pod IP: %v", err)
+		return nil, err, isIPv6
 	}
 	ipStr := ipfixCollectorIP.ipv4.String()
-	tb.Logf("Applying flow aggregator YAML with ipfix collector address: %s", ipStr)
-	if err := testData.deployFlowAggregator(fmt.Sprintf("%s:%s:tcp", ipStr, ipfixCollectorPort)); err != nil {
+	if net.ParseIP(ipStr).To4() == nil {
+		ipStr = fmt.Sprintf("[%s]", ipStr)
+		isIPv6 = true
+	}
```

Comment:
confused by this change, first because of the comment at the beginning of the function:

```go
	// TODO: remove hardcoding to IPv4 after flow aggregator supports IPv6
	isIPv6 := false
```

but also because of `ipStr := ipfixCollectorIP.ipv4.String()`, which would imply that `ipStr` is an IPv4 address

Reply:
Yes did not realize the comment is there. Yes there is a bug. I tried to carry over the existing v6 code on master, but missed a line. Will fix.